exp-player/addon/styles/components/exp-overview.scss

```diff
@@ -0,0 +1,5 @@
+.format-average-label {
```

Comment:
Table cell widths are more a suggestion, and text may still overflow positions if a single word (without spaces) is unusually long. At some point I really strongly encourage you to try a few likely situations outside the normal as outlined- not just making it good in english on a big monitor, but deliberately trying to break the assumptions behind your layout and seeing what happens.

In particular, I'm worried that we have some table cells that break if a word or span is missing. That seems like it could be very sensitive to word lengths.